src/Components/CollectionsRail/CollectionsRail.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Col, Grid, Row, Sans, Spacer } from "@artsy/palette"
+import { CollectionsRail_collections } from "__generated__/CollectionsRail_collections.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+
+import { CollectionEntity } from "./Collection"
+
+interface CollectionRailsProps {
+  collections?: CollectionsRail_collections
+  // collections?: any
+}
+
+const RailsWrapper = styled(Box)`
+  width: 100%;
+  max-width: 1250px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+`
+
+const CollectionsRow = styled(Row)`
+  justify-content: space-between;
+`
+
+export class CollectionsRail extends React.Component<CollectionRailsProps> {
+  getRails = () => {
+    const collections: any[] = this.props.collections.map(
+      (collection, index) => {
+        return (
+          <Col xl={5} lg={5} md={5} sm={12} key={index}>
```

Comment:
In the Palette example it looks like you should be able to declare padding on the column, and leave its width at 6 or 50% -- https://palette.artsy.net/tokens/grid/
Would keep investigating this as it would allow you to remove the `CollectionsRow` above, and check with frontend-platform if it is not acting as expected.

Reply:
It looks like that's the case for individual columns so I would only want to do that on index 0 and 2. I guess I could conditionally do that. I'll give that a try.

Reply:
Yah, ideally the whole grid can go and we can rely on just flexbox and our responsive styles

Reply:
@damassi this @xtina-starr and I were discussing while pairing on this, is it preferable to use `Flex` and `Box` over `Grid` generally?

Reply:
Each has its proper place, but I would lean (unless a grid for whatever reason makes total sense) on flex box just because in the end it's one less external dependency to worry about.